style.css

```diff
@@ -0,0 +1,182 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    min-width: 100vw;
+    min-height: 100vh;
+    position: relative;
+    left: 0;
+    top: 0;
+    background-color: lightblue;
+}
+
+/* header  */
+.heading {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    text-align: center;
+    padding: 12px;
+    background-color: darkblue;
+    color: ivory;
+}
+
+.addCardBtn {
+    position: fixed;
+    left: 50%;
+    top: 60px;
+    transform: translateX(-50%);
+    display: block;
+    border: 2px solid darkblue;
+    box-shadow: inset 0 0 1em gold;
+    margin: 0 auto;
+    background-color: blue;
+    font-size: 20px;
+    padding: 10px 20px;
+    border-radius: 2px;
+}
+
+.popup {
+    display: none
+}
+
+.popupDisplay {
+    display: block;
+    background-color: rgb(0, 0, 0, .5);
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+}
+
+/* popup form */
+.addCard {
+    height: 150px;
+    width: 25vw;
+    background-color: #8CABF5;
+    font-size: 2rem;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.closeBtn {
+    position: absolute;
+    top: 0;
+    right: 0;
+    font-size: 1rem;
+    color: black;
+}
+
+.addCardElement {
+    text-align: center;
+}
+
+.addCard__input {
+    width: 100%;
+    height: 20%;
+    padding: 3px 10px;
+}
+
+/* Added Card */
```

Comment:
remove comment and rename class if you think it's necessary to be more descriptive there